sway-lsp/src/utils/test.rs

```diff
@@ -16,8 +20,8 @@ pub(crate) fn e2e_test_dir() -> PathBuf {
         .join("struct_field_access")
 }
 
-pub(crate) fn sway_example_dir() -> PathBuf {
```

Comment:
This wasn't being used

Reply:
Can we put this function back in? Although it's not being used in any current CI tests, it is handy to have when testing locally.